core-services/release-controller/_repos/origin-4.3-fedora.repo

```diff
@@ -0,0 +1,32 @@
+[fedora]
+name=Fedora 31 - x86_64
+metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-31&arch=x86_64
+enabled=1
+metadata_expire=7d
+repo_gpgcheck=0
+type=rpm
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-31-x86_64
+skip_if_unavailable=False
+
+[updates]
+name=Fedora 31 - x86_64 - Updates
+metalink=https://mirrors.fedoraproject.org/metalink?repo=updates-released-f31&arch=x86_64
+enabled=1
+metadata_expire=7d
+repo_gpgcheck=0
+type=rpm
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-31-x86_64
+skip_if_unavailable=False
+
+[updates-testing-modular]
```

Comment:
TODO: replace with `modular` and `updates-modular` when https://bodhi.fedoraproject.org/updates/FEDORA-MODULAR-2019-24636b4ff9 lands